Add welcome email to Portland 2021 (#1388)
writethedocs/www: docs/conf/portland/2021/news/welcome.rst

:template: {{year}}/generic.html

.. post:: Nov 23, 2020
   :tags: {{shortcode}}-{{year}}, website, tickets

Announcing Write the Docs Portland {{year}}
===========================================

Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/portland/{{year}}/>`_,
our fourth virtual conference.
We have been tuning our approach to these events since our first virtual event in August,

> and are excited to invite you back to our online space.

Reviewer: ```suggestion
and are happy to invite you back to our online space.
```

Reviewer: Too much excitement ;)

We're excited to announce the dates for our next conference:

> **Write the Docs Portland 2021 will be April 25-27, 2021**

Reviewer: ```suggestion
**Write the Docs Portland 2021 will be on April 25-27, 2021**
```

It's been a rough year for a lot of us,
and we are still putting the finishing touches on the event this year.

> We plan to have more detail over the next couple weeks,

Reviewer: ```suggestion
We plan to have more details during the next few weeks,
```

but there's a few more detailed updates below.

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

We plan to launch our CFP in early December and have it last until sometime in January.
Get started coming up with ideas for the conference now.

Tickets
-------

Tickets will officially go on sale in **January {{ year }}**.
We will announce ticket prices soon,
so that you can start planning your budget.
As with the previous years, we have tiered pricing to make it fair for everyone.

As always, we keep ticket prices low so that it is accessible to the widest range of people possible.

> However, if you can't afford these prices and still wish to attend, please email us at portland@writethedocs.org, and we can see how we can help.

Reviewer: Should we mention the grant program here?

Sponsorship
-----------

> We are planning to do similar sponsorship levels again this year as last year.

Reviewer: ```suggestion
We are planning to do similar sponsorship levels again this year as last year. In addition to the regular sponsorship levels, we are working on additional ways to sponsor our online events.
```

> If you're interested in sponsoring the conference,

Reviewer: ```suggestion
In the meantime, 
```
